.github/pull_request_template.md

```diff
@@ -23,6 +23,18 @@ What's changed:
 - Lorem ipsum dolor sit amet
 - Consectetur adipiscing elit
 
+## Tested Scenarios
+
+- Lorem ipsum dolor sit amet
+- Consectetur adipiscing elit
+
+<details>
+<summary>Detailed scenario</summary>
+
+1. Lorem ipsum dolor sit amet
+2. Consectetur adipiscing elit
+</details>
```

Comment:
Should this be nested within a list? Refer to PR description for example.

Reply:
Lets not nest it inside a list, feels pretty broken (at least in Safari on my end)